core/src/packet_deserializer.rs

```diff
@@ -13,8 +13,8 @@ use {
 pub type BankingPacketBatch = (Vec<PacketBatch>, Option<SigverifyTracerPacketStats>);
 pub type BankingPacketReceiver = CrossbeamReceiver<BankingPacketBatch>;
 
-/// Results from deserializing packet batches.
-pub struct ReceivePacketResults {
+/// Deserialized packets and associated metadata
```

Comment:
Mmmm, can't decide if `Info` is more accurate than `Results` in describing the structure purpose. Its intention seems to pack metrics counters related to receiving/deserializing together with result `deserialized_packets`.

Reply:
Main thing is the comment I left in the description:
> Using Result in the struct name may imply it's a rust Result.

Info might not be the best either, but I think it's better than Result